client/src/App.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
```

Comment:
We might want to edit this in the public/index.html page and then we can also remove the existing font input that is not used